third-party/components/ApprovalCommentForm/ApprovalCommentForm.scss

```diff
@@ -0,0 +1,137 @@
+@import '../../../src/variables';
```

Comment:
Should this copy of these styles be prefixed? We don't want to override anything that's happening in the actual sidebar.

Reply:
This is the 3rd party CSS so it'll happen again once I use the UI elements/BRUI version of ACF but i'll prefix it here until then 👍